src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,206 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.Temporal;
+import java.util.Scanner;
+
+import duke.datetime.DateTime;
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Responsible for file-related operations, such as reading and writing of the data file when dealing with loading
+ * and saving of task list.
+ */
+public class Storage {
+    //CHECKSTYLE.OFF: Checker
+    /** Absolute file path where the folder containing the data file is stored in. */
+    private String data_file_folder_path;
+
+    /** Absolute file path where the data file should be stored in. **/
+    private String data_file_path;
```

Comment:
This naming style does not follow the convention. Can you explain the reason behind this?

Reply:
Sorry, my bad. Must have been doing too much Python...